spec/services/ga_tracker_spec.rb

```diff
@@ -167,34 +167,6 @@
     end
   end
 
-  describe '#send_request_event' do
-    context "when the visit was created" do
-      before do
-        switch_feature_flag_with :ga_id, web_property_id
-      end
-
-      it 'sends an event', vcr: { cassette_name: 'requested_visit_event' } do
```

Comment:
You could also delete the VCR cassette `spec/fixtures/vcr_cassettes/requested_visit_event.yml`

Reply:
Deleted